StreamChatClient_v3/Controllers/ChannelListController_Tests.swift

```diff
@@ -81,7 +81,7 @@ class ChannelListController_Tests: StressTestCase {
         controller.startUpdating()
         
         // Assert the existing channel is loaded
-        XCTAssertEqual(controller.channels.map { $0.cid }, [cidMatchingQuery])
+        XCTAssertEqual(controller.channels.map(\.cid), [cidMatchingQuery])
```

Comment:
This is not related to `Add setup for `ChannelListController` tests`

Reply:
It is SwiftFormat change

Reply:
You probably ran it manually, because it's set only as a pre-commit hook. It doesn't format code that's not staged.